Utilities/FourCC.cs

```diff
@@ -0,0 +1,33 @@
+﻿namespace UnityEngine.Experimental.Input.Utilities
+{
+    /// <summary>
+    /// Four character code.
+    /// </summary>
+    public struct FourCC
```

Comment:
i like how compact `FourCC` is but as a type name it's not very obvious - can we do `FourCharacterCode` ?